FightPandemics/Features/Shared/PillButton/ButtonPrototypeController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ButtonPrototypeVC.swift
+//  FightPandemics
+//
+//  Created by Luciano Schillagi on 5/16/20.
+//
+//  Copyright (c) 2020 FightPandemics
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import UIKit
+
+class ButtonPrototypeController: UIViewController {
+
+    private var regularButton = PillButton()
```

Comment:
thank you for the quick turnaround & addressing of feedback! in terms of consolidating into 1 button - i maybe miscommunicated 😅 

what we're going for is something like

```
enum PillButtonType {
    case .regular
    case .outline
    case .bold
}
```

and the ability to configure the PillButton as such, e.g. 

```
let postButton = PillButton(type: .regular, title: "PostCTA".localized)
postButton.active = false
```

Reply:
Ok